.ci_support/linux_64_c_compiler_version11cuda_compilernvcccuda_compiler_version11.8cxx_compiler_version11numpy1.22python3.10.____cpython.yaml

```diff
@@ -0,0 +1,39 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '11'
+cdt_name:
+- cos7
+channel_sources:
+- conda-forge
+channel_targets:
+- conda-forge openmm_rc
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+- '11.8'
```

Comment:
If I look at [this CI build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=853152&view=logs&jobId=3a25022a-f08c-5c04-be7d-d279a7fc48ef&j=d1321064-f6c3-56b7-0172-6d994d01c836&t=39022612-2533-571e-7b41-94814e7d38bb ), it points to this variant file

Looking at the `cuda_compiler` above it show it as `nvcc`, which it should as it is CUDA 11 (not `cuda-nvcc` as used by CUDA 12 or in the error message). This appears to be the case for the rest of the variants as well

Is something else setting `cuda_compiler` later? Could we try re-rendering with the latest changes again?

Reply:
How do I do that?  Just create a new empty PR and tell it to rerender?